src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,31 @@
-import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
+import { Comment } from '../../types/Comment';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
-);
+export type PostListProps = {
+  posts: Post[],
+  comments: Comment[];
+};
+
+export const PostList: React.FC<PostListProps> = ({ posts, comments }) => {
```

Comment:
This component should just receive Posts -> prepared posts with appropriate comments. Generally we are striving for Container -> Presentational component pattern (link [here](https://www.patterns.dev/posts/presentational-container-pattern)) where it should be App.tsx concern to prepare the data accordingly and this components only responsibility should be to display them. Functions that do the mapping in the example I showed you in App.tsx should do the trick

Reply:
I removed comments from list.